test-packages/test-services/openapi/test-service/api.ts

```diff
@@ -12,14 +12,24 @@ export const TestServiceApi = {
     stringParameter?: string,
     integerParameter?: number,
     dollarParameter?: string,
-    dotParameter?: string
+    dotParameter?: string,
+    enumStringParameter?: 'value1' | 'value2',
+    enumInt32Parameter?: 1 | 2,
+    enumFloatParameter?: 1 | 2,
+    enumDoubleParameter?: 1 | 2,
```

Comment:
Consider whether we really need all three cases for number - seems trivial to me and it makes the definition of the testentity type more complex, maybe one or two are enough? I have no strong opinion here, up to you.

Reply:
I would like to keep two: `int` and `double` so we know both of these are tested, while `float` might be useless.